src/components/common/YourCompany/YourCompany.vue

```diff
@@ -467,7 +466,7 @@ export default class YourCompany extends Mixins(
       return this.getResource.changeData.nameChangeOptions
         .filter(option => option !== CorrectionTypes.CORRECT_NAME_TO_NUMBER)
     }
-
+    if (this.isEntityTypeFirm) return Array(CorrectionTypes.CORRECT_NEW_NR)
```

Comment:
You should use the resource files to do this, not special-case code.

We want the components to be as generic as possible. We have resource files to specify a lot of things.

Reply:
This is now changed.